src/shipit_signoff/shipit_signoff/api.py

```diff
@@ -183,40 +180,85 @@ def sign_off(uid):
 
     claim_group = request.json['group']
 
-    # TODO: is the claim_group in the policy for step uid?
-
     if not is_user_in_group(claim_group):
         abort(403)
 
     existing_signatures = step.signatures
     email = get_logged_in_email()
-    # TODO Process the definition only if 'local' is set
+
     policy_definition = step.policy_data['definition']
 
+    if step.policy_data['method'] == 'balrog':
+        abort(400, 'Balrog method not yet implemented')
+    if step.policy_data['method'] != 'local':
```

Comment:
Totally a style thing (feel free to ignore), but this might read better as:
```
if method == 'balrog':
    ...
elif method == 'local':
    ...
else:
    ...
```

I find that mixing == and != in a series of conditions comparing the same value is a bit confusing.